lib/user.js

```diff
@@ -99,12 +100,25 @@ User.prototype.anonymousId = function(anonymousId) {
   // set / remove
   if (arguments.length) {
     store.set('ajs_anonymous_id', anonymousId);
+    localStorage.set('ajs_anonymous_id', anonymousId);
     return this;
   }
 
   // new
   anonymousId = store.get('ajs_anonymous_id');
   if (anonymousId) {
+    // value exist in cookie, copy it to localStorage
+    localStorage.set('ajs_anonymous_id', anonymousId);
```

Comment:
@f2prateek this line is new (was not in the original code you reviewed.) Please have another look!

Reply:
Yep I saw this -  this makes sense to me!